components/layout_2020/query.rs

```diff
@@ -437,9 +444,36 @@ fn process_offset_parent_query_inner(
     // zero and terminate this algorithm." (others)
     let node_offset_box = node_offset_box?;
 
-    let offset_parent_padding_box_corner = node_offset_box
-        .offset_parent_node_address
-        .map(|offset_parent_node_address| {
+    let offset_parent_padding_box_corner = if let Some(offset_parent_node_address) =
+        node_offset_box.offset_parent_node_address
+    {
+        // Although the spec (https://www.w3.org/TR/cssom-view-1/#extensions-to-the-htmlelement-interface)
+        // says that offsetTop/offsetLeft are always relative to the padding box of the offsetParent.
+        // In practice this is not true in major browsers in the case that the offsetParent is the body
```

Comment:
The 1st sentence ends abruptly.

```suggestion
        // The spec (https://www.w3.org/TR/cssom-view-1/#extensions-to-the-htmlelement-interface)
        // says that offsetTop/offsetLeft are always relative to the padding box of the offsetParent.
        // However, in practice this is not true in major browsers in the case that the offsetParent is the body
```